### PR TITLE
Yessenia Rodriguez css-brochure-form

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-01-20 at 6 47 23 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  background-color: #f25c54;
+  border-radius: 2.75em;
+  color: white;
+  border: none;
+  font-family: 'Brandon Grotesque', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  width: 130px;
+  height: 50px;
+  margin-left: 470px;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:67]: Woah, that's a lot of margin!! If you are doing this to place the button on the right side of the form, you should just apply `text-align: right` to the `div` that the `button` is a child of.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nicely done, Yessenia!! One minor change:

File: css-brochure-form/styles.css (hunk 1 above)
Comment (yesseniarod) [reply css-brochure-form/styles.css]: Screen Shot 2021-01-22 at 5 29 23 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**